precompile/allow_list.go

```diff
@@ -31,17 +31,11 @@ var (
 	allowListInputLen = common.HashLength
 )
 
-// AllowListConfig specifies the configuration of the allow list.
-// Specifies the block timestamp at which it goes into effect as well as the initial set of allow list admins.
+// AllowListConfig specifies the initial set of allow list admins.
 type AllowListConfig struct {
-	BlockTimestamp *big.Int `json:"blockTimestamp"`
```

Comment:
Could anything weird happen by removing this?

Reply:
I don't think so. The replacement `UpgradeableConfig` is embedded in the precompile configs similar to how `AllowListConfig` is embedded, so this change only moves it.